raiden_contracts/deploy/__main__.py

```diff
@@ -170,12 +170,18 @@ def raiden(
     secret_registry_from_deployment_file: Optional[str],
 ) -> None:
     check_version_dependent_parameters(contracts_version, max_token_networks)
+    if secret_registry_from_deployment_file is not None:
+        secret_registry_from_deployment_path: Optional[Path] = Path(
```

Comment:
This could be handled in a nicer way by user a click Path option: https://click.palletsprojects.com/en/7.x/arguments/#file-path-arguments.

Reply:
That's nice.  I'll do this in this PR.